Dockerfile

```
@@ -6,7 +6,7 @@ ARG TARGETARCH
 WORKDIR /namespace-lister
 
 # Copy the Go Modules manifests
-COPY go.mod go.sum .
+COPY go.mod go.sum ./
```

I don't see a difference in behavior - why is this needed?

---

I applied a fix for a complaint from my linter. When multiple files are copied, it's best practice to explicitly declare a folder.

---

Then maybe we should have something like this?
```suggestion
COPY go.mod go.sum /namespace-lister/
```

---

we are already setting the WORKDIR to `/namespace-lister`. I'd prefer this to remain relative to the WORKDIR